01-intro.Rmd

```diff
@@ -88,7 +88,7 @@ workflow my_workflow {
     
     call do_something {
         input:
-            fastq = fq,
+            fastq = fq,  ## example of referring to workflow-level input
```

Comment:
I like these comments, but I think they need to be more consistent in formatting. At line 78 and 91 they're `## formatted like this` but in 114 and 132 they're `#like this instead`. I'd recommend `# this format` personally but it's sort of a matter of preference.

Reply:
Fair enough. I'll be more consistent in the future